odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -210,10 +212,40 @@ def compute_auto_rotation_backup_dts(self, backup_dts, hourly=0, daily=0, weekly
                         break
         return needed_backup_dts
 
+    @api.model
+    def get_simulation_backup_list(self, config):
+        info_records = self.env['odoo_backup_sh.backup_info'].search([('backup_simulation', '=', True),
+                                                                      ('storage_service', '=', config.storage_service),
+                                                                      ('database', '=', config.database)])
+        simulation_backup_list = []
+        for r in info_records:
+            simulation_backup_list.append(('%s.%s.info' % (r.database,
+                                                           r.upload_datetime.strftime(REMOTE_STORAGE_DATETIME_FORMAT)),
+                                           config.storage_service))
+            simulation_backup_list.append(('%s.%s.zip' % (r.database,
+                                                          r.upload_datetime.strftime(REMOTE_STORAGE_DATETIME_FORMAT)),
+                                           config.storage_service))
+        return simulation_backup_list
+
     @api.model
     def get_backup_list(self, cloud_params):
         try:
-            return BackupCloudStorage.get_backup_list(cloud_params)
+            backup_list = BackupCloudStorage.get_backup_list(cloud_params)
+            # if the simulation configuration exists, we need to get
+            # all the simulation info files and convert them to backup format
+            domain = [('storage_service', '=', 'odoo_backup_sh'), ('backup_simulation', '=', True), ('database', '=', self._cr.dbname)]
+            # first search with rotation
+            config = self.env['odoo_backup_sh.config'].search(domain)
+            # if the config does not exist, we need to find the config without rotation
+            if not config:
+                domain.append(('active', '=', False))
+                config = self.env['odoo_backup_sh.config'].search(domain)
```

Comment:
the `active` field - is compute filed: `backup_config.config_cron_ids and any(backup_config.config_cron_ids.mapped('active'))`

there may be such cases:

We created fake backup records but the config don't have rotations, then if we click to "Update Info" or "Make Backup" buttons all fake records will removed because the search method don't find this config and we cannot check the backup_simulation config (because active is False)

Reply:
with_context({'active_test': False})